app/services/process_feed_service.rb

```diff
@@ -50,6 +49,7 @@ def create_status
       if verb == :share
         original_status, = status_from_xml(@xml.at_xpath('.//activity:object', activity: TagManager::AS_XMLNS))
         status.reblog    = original_status
+        status.text      = ''
```

Comment:
what's this addition doing? fixing null text for reblogged entries? is this necessarily part of the fix?